src/contracts/misc/GhoAaveSteward.sol

```diff
@@ -0,0 +1,155 @@
+import {ReserveConfiguration} from '@aave/core-v3/contracts/protocol/libraries/configuration/ReserveConfiguration.sol';
+import {GhoInterestRateStrategy} from '../facilitators/aave/interestStrategy/GhoInterestRateStrategy.sol';
+import {IFixedRateStrategyFactory} from '../facilitators/aave/interestStrategy/interfaces/IFixedRateStrategyFactory.sol';
+import {IGhoAaveSteward} from './interfaces/IGhoAaveSteward.sol';
+import {RiskCouncilControlled} from './RiskCouncilControlled.sol';
+
+/**
+ * @title GhoAaveSteward
+ * @author Aave Labs
+ * @notice Helper contract for managing parameters of the GHO reserve
+ * @dev Only the Risk Council is able to action contract's functions, based on specific conditions that have been agreed upon with the community.
+ */
+contract GhoAaveSteward is Ownable, IGhoAaveSteward, RiskCouncilControlled {
+  using ReserveConfiguration for DataTypes.ReserveConfigurationMap;
+
+  /// @inheritdoc IGhoAaveSteward
+  uint256 public constant GHO_BORROW_RATE_CHANGE_MAX = 0.0500e27; // 5.00%
+
+  /// @inheritdoc IGhoAaveSteward
+  uint256 public constant GHO_BORROW_RATE_MAX = 0.2500e27; // 25.00%
+
+  /// @inheritdoc IGhoAaveSteward
+  uint256 public constant MINIMUM_DELAY = 2 days;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable POOL_ADDRESSES_PROVIDER;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable GHO_TOKEN;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable FIXED_RATE_STRATEGY_FACTORY;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable RISK_COUNCIL;
+
+  GhoDebounce internal _ghoTimelocks;
+
+  /**
+   * @dev Only methods that are not timelocked can be called if marked by this modifier.
+   */
+  modifier notTimelocked(uint40 timelock) {
+    require(block.timestamp - timelock > MINIMUM_DELAY, 'DEBOUNCE_NOT_RESPECTED');
+    _;
+  }
+
+  /**
+   * @dev Constructor
+   * @param owner The address of the owner of the contract
+   * @param addressesProvider The address of the PoolAddressesProvider of Aave V3 Ethereum Pool
+   * @param ghoToken The address of the GhoToken
+   * @param fixedRateStrategyFactory The address of the FixedRateStrategyFactory
+   * @param riskCouncil The address of the risk council
+   */
+  constructor(
+    address owner,
+    address addressesProvider,
+    address ghoToken,
+    address fixedRateStrategyFactory,
+    address riskCouncil
+  ) RiskCouncilControlled(riskCouncil) {
+    require(owner != address(0), 'INVALID_OWNER');
+    require(addressesProvider != address(0), 'INVALID_ADDRESSES_PROVIDER');
+    require(ghoToken != address(0), 'INVALID_GHO_TOKEN');
+    require(fixedRateStrategyFactory != address(0), 'INVALID_FIXED_RATE_STRATEGY_FACTORY');
+    require(riskCouncil != address(0), 'INVALID_RISK_COUNCIL');
+
+    POOL_ADDRESSES_PROVIDER = addressesProvider;
+    GHO_TOKEN = ghoToken;
+    FIXED_RATE_STRATEGY_FACTORY = fixedRateStrategyFactory;
+    RISK_COUNCIL = riskCouncil;
```

Comment:
This is done within `RiskCouncilControlled` right?

Reply:
Updated (see below comment)